### PR TITLE
minimum effort ie11 fixes

### PR DESCRIPTION
**Pull request by partouf**
Similar to #1498

### CODE REVIEW

File: static/compiler-service.js [hunk 1]
```diff
@@ -31,7 +31,7 @@ var Promise = require('es6-promise').Promise;
 
 function CompilerService() {
     this.base = window.httpRoot;
-    if (!this.base.endsWith('/')) {
```
Comment (RabsRincon) [inline static/compiler-service.js]: Probably better to use _.endsWith, right? Im not sure how this would behave for empty bases

Comment (partouf) [reply static/compiler-service.js]: yes, but i don't think that exists, i saw it's in lodash, but its not in underscore, and i think we're using underscore?

**Comment by mattgodbolt** [comment]
On balance I think I mildly prefer the polyfill, but only as it's slightly less changes. (even though it introduces a magic shim thing...)

**Comment by partouf** [comment]
> On balance I think I mildly prefer the polyfill, but only as it's slightly less changes. (even though it introduces a magic shim thing...)

I agree

**Pull request closed without merging**